qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -1320,6 +1320,30 @@ def update_category(self, category, samples_and_values):
 
                 raise e
 
+    def get_category(self, category):
+        """Returns the values of all samples for the given category
+
+        Parameters
+        ----------
+        category : str
+            Metadata category to get information for
+
+        Returns
+        -------
+        dict
+            Sample metadata for the category in the form {sample_id: value}
+
+        Raises
+        ------
+        ValueError
```

Comment:
This error is not raised here. At the same time, I would argue that the error raised here should be a subtype of QiitaDB. I would prefer to follow the structure that is done in other places, in which we first check that the column actually exists, and if it exists, we retrieve such information. This will also remove the usage of "scrub_data" which I don't think it is safe in terms that we may suffer from SQL injection.